### DISCUSSION TITLE
Does anyone uses DJL to leverage integration testing?

### DISCUSSION THREAD

**Discussion by alexandros173** [deepjavalibrary/djl#427] [Q&A] [answered]
I'm thinking of doing a demo for my client but I don't know any libraires or examples with DJL and integration testing

**Comment by lanking520** [comment, marked as answer]
Ok, to start with, you can try with our getting started kit: http://docs.djl.ai/master/docs/quick_start.html.

After that, you can play with a few existing projects we had: https://github.com/aws-samples/djl-demo. These includes a huge set of examples you can run out-of-box. As long as you have java installed, you should be able to run them successfully.
